releases/v21.2.2.md

```diff
@@ -0,0 +1,228 @@
+- CockroachDB is now less likely to OOM when queries reading a lot of data are issued with high concurrency (these queries are likely to hit the memory budget determined by [`--max-sql-memory` startup parameter](../v21.2/cockroach-start.html#general)). [#70809][#70809]
+- The `indexprs` column of [`pg_catalog.pg_index`](../v21.2/pg-catalog.html) is now populated with string representations of every expression element in the index. If the index is not an [expression index](../v21.2/expression-indexes.html), `indexprs` is `NULL`. The `indexdef` column of `pg_catalog.pg_indexes` and the `indpred` column of `pg_catalog.pg_index` now correctly display user-defined types. [#72870][#72870]
+- Fixed a bug where introspection tables and error messages would not correctly display intervals according to the `intervalstyle` [session variable](../v21.2/set-vars.html). [#72690][#72690]
+- Fixed a bug where index definitions in [`pg_catalog.pg_indexes`](../v21.2/pg-catalog.html) would not format intervals according to the `intervalstyle` [session variable](../v21.2/set-vars.html). [#72903][#72903]
+- [Statement statistics](../v21.2/ui-statements-page.html#statement-statistics) are now grouped by the statement's corresponding transaction fingerprints. [#72941][#72941]
+- The query backing [`crdb_internal.cluster_contended_indexes`](../v21.2/crdb-internal.html) improperly assumed that index IDs were unique across the database. This change adds the proper scoping by table descriptor ID, truing up the contents of that view. [#73025][#73025]
+- The timeout when checking for Raft application of upgrade migrations has been increased from 5 seconds to 1 minute, and is now controllable via the [cluster setting](../v21.2/cluster-settings.html) `kv.migration.migrate_application.timeout`. This makes migrations much less likely to fail in clusters with ongoing rebalancing activity during upgrade migrations. [#73061][#73061]
+- Fixed a bug that could cause some semi lookup joins on [`REGIONAL BY ROW` tables](../v21.2/multiregion-overview.html#regional-by-row-tables) to return early before finding all data. This bug is currently only present on 21.2.0. This problem only manifested if there was an `ON` condition on top of the equality condition used for the lookup join, and the lookup columns did not form a key in the index being looked up into. The primary impact of this issue for most users relates to uniqueness checks on mutations of `REGIONAL BY ROW` tables, since uniqueness checks are implemented with a semi lookup join with an `ON` condition. The result of this bug was that uniqueness checks were not comprehensive, and could miss an existing duplicate key on a remote node. This could cause data to be erroneously inserted with a duplicate key when it should have failed the uniqueness check. These problems have now been fixed. [#73063][#73063]
```

Comment:
This bug is already fixed in 21.2.1: https://www.cockroachlabs.com/docs/releases/v21.2.1.html#bug-fixes, so shouldn't appear here.

I'm not sure if there are any other dups with 21.2.1.

Reply:
I removed all the dups from v21.2.1.